src/Backend/Modules/Tags/Entity/TagConnection.php

```diff
@@ -0,0 +1,127 @@
+<?php
+
+namespace Backend\Modules\Tags\Entity;
+
+/*
+ * This file is part of Fork CMS.
+ *
+ * For the full copyright and license information, please view the license
+ * file that was distributed with this source code.
+ */
+
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * This is the TagConnection Entity
+ *
+ * @author Jeroen Desloovere <info@jeroendesloovere.be>
+ *
+ * @ORM\Entity
+ * @ORM\Table(name="modules_tags")
```

Comment:
Since other modules rely on the current table name, I'd think we should wait until Doctrine is fully integrated.